cicd/forgeops-tests/configure.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+import sys
+import os
+import shutil
+import yaml
+import re
+
+TESTS_NAMESPACE = "smoke"
+if "TESTS_NAMESPACE" in os.environ:
+    TESTS_NAMESPACE = os.environ["TESTS_NAMESPACE"]
+
+TESTS_DOMAIN = "forgeops.com"
+if "TESTS_DOMAIN" in os.environ:
+    TESTS_DOMAIN = os.environ["TESTS_DOMAIN"]
+
+TESTS_IMAGE_TAG = "7.0.0-SNAPSHOT"
+if "TESTS_IMAGE_TAG" in os.environ:
+    TESTS_IMAGE_TAG = os.environ["TESTS_IMAGE_TAG"]
+
+TESTS_DEPLOYMENT = "smoke-deployment"
+if "TESTS_DEPLOYMENT" in os.environ:
+    TESTS_DEPLOYMENT = os.environ["TESTS_DEPLOYMENT"]
+
+TESTS_COMPONENTS = "openam amster"
+if "TESTS_COMPONENTS" in os.environ:
+    TESTS_COMPONENTS = os.environ["TESTS_COMPONENTS"]
+
+IMAGE_BASE_URL = 'forgerock-docker-public.bintray.io/forgerock'
+IMAGE_PULL_POLICY = 'Always'
+
+OPENAM_HELM_SUBFOLDER = "openam"
+OPENAM_IMAGE_NAME = "openam"
+OPENAM_DEPS_COMPONENTS = "configstore"
+
+AMSTER_HELM_SUBFOLDER = "amster"
+AMSTER_IMAGE_NAME = "amster"
+AMSTER_DEPS_COMPONENTS = ""
+
+USERSTORE_HELM_SUBFOLDER = "ds"
+USERSTORE_IMAGE_NAME = "ds-paas"
+USERSTORE_DEPS_COMPONENTS = ""
+
+CTSSTORE_HELM_SUBFOLDER = "ds"
+CTSSTORE_IMAGE_NAME = "ds-paas"
+CTSSTORE_DEPS_COMPONENTS = ""
+
+CONFIGSTORE_HELM_SUBFOLDER = "ds"
+CONFIGSTORE_IMAGE_NAME = "ds-paas"
+CONFIGSTORE_DEPS_COMPONENTS = ""
+
+OPENIDM_HELM_SUBFOLDER = "openidm"
+OPENIDM_IMAGE_NAME = "idm"
+OPENIDM_DEPS_COMPONENTS = "postgres-openidm"
+
+OPENIG_HELM_SUBFOLDER = "openig"
+OPENIG_IMAGE_NAME = "ig"
+OPENIG_DEPS_COMPONENTS = ""
+
+
+def main(argv):
+    # root_dir : computed using relative position from this file
+    current_dir = os.path.abspath(os.path.dirname(__file__))
+    os.chdir(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', '..'))
+    root_dir = os.getcwd()
+    os.chdir(current_dir)
+
+    helm_dir = os.path.join(root_dir, 'helm')
+    config_dir = os.path.join(root_dir, 'samples', 'config', TESTS_DEPLOYMENT)
+
+    env_components = "frconfig"
+    for component in TESTS_COMPONENTS.split(" "):
+        subfolder = eval('%s_HELM_SUBFOLDER' % component.upper())
```

Comment:
isn't subfolder just the component(s) passed in TESTS_COMPNENTS??

Reply:
For openam, amster, openidm and openig yes but not for the userstore, configstore and ctsstore => for these one the helm subfolder is `ds`